README.md

```diff
@@ -8,88 +8,121 @@ Official [Buildkite Test Analytics](https://buildkite.com/test-analytics) collec
 
 ## 👉 Installing
 
-1) [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the API token that it gives you.
+1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the API token that it gives you.
 
-2) Add the [`buildkite-test-collector` package](https://www.npmjs.com/package/buildkite-test-collector):
+2. Add the [`buildkite-test-collector` package](https://www.npmjs.com/package/buildkite-test-collector):
 
-    ```bash
-    # If you use npm:
-    npm install --save-dev buildkite-test-collector
+   ```bash
+   # If you use npm:
+   npm install --save-dev buildkite-test-collector
 
-    # or, if you use yarn:
-    yarn add --dev buildkite-test-collector
-    ```
+   # or, if you use yarn:
+   yarn add --dev buildkite-test-collector
+   ```
 
-3) Add the buildkite test collector to your testing framework:
+3. Add the buildkite test collector to your testing framework:
 
-    ### Jest
+   ### Jest
 
-    Update your [Jest configuration](https://jestjs.io/docs/configuration):<br>
+   Update your [Jest configuration](https://jestjs.io/docs/configuration):<br>
 
-    ```js
-      // jest.config.js
+   ```js
+     // jest.config.js
 
-      // Send results to Test Analytics
-      reporters: [
-        'default',
-        'buildkite-test-collector/jest/reporter'
-      ],
+     // Send results to Test Analytics
+     reporters: [
+       'default',
+       'buildkite-test-collector/jest/reporter'
+     ],
 
-      // Enable column + line capture for Test Analytics
-      testLocationInResults: true
-    ```
+     // Enable column + line capture for Test Analytics
+     testLocationInResults: true
+   ```
 
-    ### Jasmine
+   If you would like to pass in the API token using a custom environment variable, you can do so using the report options.
 
-    [Add the buildkite reporter to jasmine](https://jasmine.github.io/setup/nodejs.html#reporters):<br>
+   ```js
+   // jest.config.js
 
-    ```js
-      // SpecHelper.js
-      var BuildkiteReporter = require('buildkite-test-collector/jasmine/reporter');
-      var buildkiteReporter = new BuildkiteReporter();
+   // Send results to Test Analytics
+   reporters: [
+     "default",
+     ["buildkite-test-collector/jest/reporter", { token: INSERT_TOKEN_HERE }],
```

Comment:
I'm not too familiar with the JavaScript testing frameworks so this might be silly question.

Are we asking folks to insert the environment variable name or the string of the token?

Reply:
You can set it up to either use a string of the token or the environment variable that will evaluate to the token string if that makes sense?

You can use [dotenv](https://github.com/motdotla/dotenv) within config files `require('dotenv').config()` and it could look like this:

```js
     ["buildkite-test-collector/jest/reporter", { token: process.env.TEST_ANALYTICS_JEST_TOKEN }]
```

Since it's unclear, I'll rewrite the docs to specify we recommend using the dotenv setup if that sounds good to you @gchan?

Reply:
Actually, I might rewrite the examples and tests to use dotenv so the use case is clearer

Reply:
Yeah that sounds good! We want to encourage people to use environment variables over hard coding secret strings into their code bases, so clarifying this in the docs and examples would be perfect

Reply:
Changes done! Ready for re-review 😄